lib/rubocop/yaml_duplication_checker.rb

```diff
@@ -5,7 +5,7 @@ module RuboCop
   module YAMLDuplicationChecker
     def self.check(yaml_string, filename, &on_duplicated)
       # Specify filename to display helpful message when it raises an error.
-      tree = YAML.parse(yaml_string, filename: filename)
+      tree = YAML.parse(yaml_string, filename)
```

Comment:
I think we need to make this conditional here as well.

Reply:
At least given that the specs work in all ruby versions used on CI can't we leave it like this? I doubt ruby-core would introduce a breaking change to the YAML module

Reply:
OK.